src/DashboardService.Models.Dto/Requests/Comment/Filters/GetCommentsFilter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Comment.Filters;
+
+public class GetCommentsFilter
+{
+  [FromQuery(Name = "isAscendingSort")]
+  public bool? IsAscendingSort { get; set; }
+
+  [FromQuery(Name = "isActive")]
+  public bool? IsActive { get; set; }
```

Comment:
call it IncludeDeactivated for consistency

Reply:
++